product/carbon-home/conf/identity/profile-mapping.yaml

```diff
@@ -149,3 +149,21 @@ profiles:
    readonly: false
    regex: "*"
    dataType: "email"
+ -
+  profileName: "user-identifier"
+  isAdminProfile: false
+  claims:
+  -
+   claimURI: "http://wso2.org/claims/username"
+   displayName: "Username"
+   required: true
+   readonly: true
+   verifier: "claim verifying extension"
```

Comment:
No harm in keeping it in the file. But if this is a newly added profile and have no use of the property, better remove it.